src/sentry/tempest/tasks.py

```diff
@@ -0,0 +1,178 @@
+import logging
+
+from sentry import http
+from sentry.models.projectkey import ProjectKey, UseCase
+from sentry.tasks.base import instrumented_task
+from sentry.tempest.models import TempestCredentials
+from sentry.utils import json
+
+TEMPEST_URL = ""  # FIXME: Set this value once we have it
+POLL_LIMIT = 348  # 348 every 5 min ~ 100k a day
+
+
+logger = logging.getLogger(__name__)
+
+
+@instrumented_task(
+    name="sentry.tempest.tasks.poll_tempest",
+    queue="tempest",
```

Comment:
Here we miss a few settings mentioned here: https://github.com/getsentry/sentry/pull/82454/files#r1895521917